index.js

```diff
@@ -1207,6 +1246,9 @@ ArrayTome.prototype.shift = function () {
 		diff(this, 'shift');
 	}
 
+	// update root
+	setRoot(out, out);
+
 	return out ? out.valueOf() : out;
```

Comment:
Tome.conjure doesn't seem to be fixing the root, I still get updates pushed to the original array. Unless you mean using `Tome.untome` ?